docs/migration.md

```diff
@@ -293,19 +293,30 @@ Before migrating the volumes make sure the following prerequisites are taken car
  - The jiva-operator should be installed with version 2.7.0 or above. You can install the correct version of jiva-operator from [charts](https://github.com/openebs/charts/tree/gh-pages). Get the jiva-operator yaml within the correct versioned folder and install. 
  - **The application needs to be scaled down before migrating.** This is required as the old PVC and PV spec needs to be deleted at the end of migration.
 
- For this example lets say the original volume has PVC name `demo-vol-claim` and PV name `pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb`.
+ For this example lets say the original volume has PVC name `demo-vol-claim` in `default` namespace and PV name `pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683`.
  
 ### Migration steps
 
-1. Scale down volume controller and all replica deployments. You can find these deployments using the command:
+1. Note down the nodes on which the replicas were scheduled originally 
+   ```
+    $ k get pods -o wide
```

Comment:
we can use pv-name label just to differentiate with other pods on per volume bases , helpful for the user to understand the no of pods ?